wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/component/plugin/PluginComponentToWorkspaceApplier.java

```diff
@@ -74,6 +75,14 @@ public void apply(
     }
 
     String pluginIdVersion = resolveIdAndVersion(pluginComponent.getId());
+    String memoryLimit = pluginComponent.getMemoryLimit();
+    if (memoryLimit != null) {
+      String pluginIdPart = pluginIdVersion.split(":")[0];
```

Comment:
It would be kind to rework this code to use PluginFQN https://github.com/eclipse/che/pull/13056#discussion_r272299332
I'm OK with the current version and I will rework it later if you do not handle it within this PR

Reply:
Public method in `PluginFQNParser` didn't looks very useful for this given case (parsing each plugin FQN separately). Am i missing something ?

Reply:
Oh. It is needed to contribute to `PluginFQNParser`. Do not mind. Will rework it later.